Assets/Scripts/Bomb.cs

```diff
@@ -4,22 +4,26 @@
 
 public class Bomb : Enemy
 {
+    // These are only initial values for behaviour construction
     private const float _MAX_SPEED = 4;
     private const float _ACCELERATION_RATE = 1;
-    private Rigidbody2D _myRigidBody, _player;
+    
+    private Rigidbody2D _player;
 
+    // Bomb does nothing while idle, approaches player otherwise
     protected override void Start(){
-        _myRigidBody = this.GetComponent<Rigidbody2D>();
-        _player = GameObject.FindGameObjectWithTag("Player").GetComponent<Rigidbody2D>();
+        this._myRigidBody = this.GetComponent<Rigidbody2D>();
+        this._player = GameObject.FindGameObjectWithTag("Player").GetComponent<Rigidbody2D>();
         
         this._idleBehaviour = new DoNothing(this, ENEMY_STATE.idle);
-        this._awareBehaviour = new ApproachTarget(this, _player, _MAX_SPEED, _ACCELERATION_RATE, ENEMY_STATE.aware);
-        this._engagedBehaviour = new ApproachTarget(this, _player, _MAX_SPEED, _ACCELERATION_RATE, ENEMY_STATE.engaged);
+        this._awareBehaviour = new ApproachTarget(this, _player, Bomb._MAX_SPEED, Bomb._ACCELERATION_RATE, ENEMY_STATE.aware);
+        this._engagedBehaviour = new ApproachTarget(this, _player, Bomb._MAX_SPEED, Bomb._ACCELERATION_RATE, ENEMY_STATE.engaged);
```

Comment:
I'm surprised `Bomb._{MAX_SPEED, ACCELERATION_RATE}` works granted they're not static variables. Shouldn't we use `this._XXX` instead?

Reply:
> I'm surprised `Bomb._{MAX_SPEED, ACCELERATION_RATE}` works granted they're not static variables. Shouldn't we use `this._XXX` instead?

they're const, which means they're tied to the class rather than the object.